README.md

```diff
@@ -5,9 +5,48 @@
 
 ## Overview
 
-The python package `libpyvinyl` exposes the high level API for simulation codes under
-the umbrella of the Virtual Neutron and x-raY Laboratory (ViNYL).
+Simulations provide invaluable insights to plan (before) and
+ understand (after) experiments at neutron and X-ray facilities. 
+A wide set of libraries and programs are already available to simulate
+ neutron and X-ray beams, propagate them through the beamlines,
+ interact with matter of a sample and get data acquired by detectors. 
+
+The aim of this library is to provide a high level API to allow users
+ to access simulations with a unified interface, reaching almost
+ seamless interoperability, allowing to chain multiple
+ simulation softwares in a natural way. 
+The vast differences with respect to parameter
+ names, unit conventions, configuration syntax, i.e. the user
+ interface, is, hence, overcome creating a `libpyvinyl` complaiant API
+ for each simulation software. 
+
+## Software specific APIs based on libpyvinyl
+The python package `libpyvinyl` provides a way to harmonize the user interfaces of such simulation codes. It is an object oriented library; its classes define the user interface to simulation codes, simulation parameters and simulation data.
 
+For a given simulation code, e.g. propagation of neutron or photon beams through a beamline, a new class would have to be defined that derives from the
+`libpyvinyl` classes. Implementing the methods meant to configure a simulation, launch the simulation code, and retrieve the output data. Since the
```

Comment:
This  derived class requires the implementation of certain methods meant to ...

Reply:
ok